src/manifest.json

```diff
@@ -1,29 +1,35 @@
 {
-  "name": "Google Tasks (by Google)",
-  "version": "1.0",
-  "description": "Official Chrome Extension for Google Tasks",
-  "browser_action": {
-    "default_icon": "images/tasks-19x19.png",
-    "default_title": "Google Tasks",
-    "popup": "popup.html"
-  },
-  "background_page": "background.html",
-  "omnibox": {
-    "keyword": "t"
-  },
-  "icons": {
-    "16": "images/tasks-16x16.png",
-    "32": "images/tasks-32x32.png",
-    "48": "images/tasks-48x48.png",
-    "128": "images/tasks-128x128.png"
-  },
-  "permissions": [
-    "contextMenus",
-    "notifications",
-    "tabs",
-    "http://www.google.com/",
-    "https://www.google.com/",
-    "http://www.googleapis.com/",
-    "https://www.googleapis.com/"
-  ]
+   "background": {
```

Comment:
I’m not sure the sort helps here. The original order was better since it kept the important bits at the top.